app/Http/Controllers/MessagesController.php

```diff
@@ -92,16 +92,20 @@ public function sendMessage(Message $message)
     {
         // Get competition with activity.
         $competitionId = request('competition_id');
-        $competition = Competition::find($competitionId);
+        $competition = null;
```

Comment:
had to set this to NULL otherwise the code below errors out because $competition doesnt exist

https://github.com/DoSomething/gladiator/blob/master/app/Http/Controllers/MessagesController.php#L119